io.github.pemsley.coot.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.1.10
+Name=Coot
+GenericName=Crystallographic Object-Oriented Toolkit
+Comment=Software for macromolecular model-building
+Exec=coot %F
+Icon=io.github.pemsley.coot
+Terminal=false
+Type=Application
+Categories=Science;Biology;Chemistry;Gtk
```

Comment:
these aren't valid categories, you can't pick at random. There is a spec for that https://specifications.freedesktop.org/menu-spec/latest/category-registry.html.

Reply:
place them as keywords and put some valid categories.

Reply:
`GTK` is OK?

Reply:
@bbhtt

Thank you for giving the link. I'll fix it.